SharedProjects/Babylon2GLTF/GLTFExporter.Material.cs

```diff
@@ -387,9 +381,8 @@ private void ExportMaterial(BabylonMaterial babylonMaterial, GLTF gltf)
                     }
                 }
             }
-            else if (babylonMaterial.GetType() == typeof(BabylonPBRMetallicRoughnessMaterial))
+            else if (babylonMaterial is BabylonPBRMetallicRoughnessMaterial babylonPBRMetallicRoughnessMaterial)
```

Comment:
Cleanup is appreciated, but its usually best to keep PRs as specific as possible, Its never an issue to make another PR to address another (seemingly) unrelated fault.